pkg/doc/jose/encrypter.go

```diff
@@ -232,35 +242,42 @@ func (je *JWEEncrypt) wrapCEKForRecipients(cek, apu, apv, aad []byte,
 	return recipientsWK, singleRecipientAAD, nil
 }
 
-func (je *JWEEncrypt) getWrapKeyOpts() (string, []cryptoapi.WrapKeyOpts) {
-	var wrapOpts []cryptoapi.WrapKeyOpts
+func (je *JWEEncrypt) encodeAPUAPVForGoJose(kek *cryptoapi.RecipientWrappedKey) {
+	if je.ForGoJoseInterop {
```

Comment:
what breaks in go-jose when this isn't done?

Reply:
My guess is that the base64url encoding needs to be fixed in AFG.

Reply:
yes, APU/APV must be base64URL encoded, in this case they're KID or epk.x which are already base64URL values, they require an additional encoding to be compliant with their definition as per the [JWA](https://tools.ietf.org/html/rfc7518#section-4.6.1.2) ietf spec.